lib/chai/utils/addProperty.js

```diff
@@ -36,7 +36,7 @@ var transferFlags = require('./transferFlags');
  */
 
 module.exports = function addProperty(ctx, name, getter) {
-  getter = getter === undefined ? new Function() : getter;
+  getter = getter === undefined ? function(){} : getter;
```

Comment:
This is very nitpicky, but for the sake of consistency with other parts of the code, could you use `function () {}` instead please? Thanks 😊